src/main/java/com/mraof/minestuck/world/lands/terrain/EndLandType.java

```diff
@@ -38,7 +38,7 @@ public class EndLandType extends TerrainLandType
 	
 	public EndLandType()
 	{
-		super(new Builder(() -> MSEntityTypes.NAKAGATOR).names(END, DIMENSION)
+		super(new Builder(MSEntityTypes.NAKAGATOR::get).names(END, DIMENSION)
```

Comment:
You should be able to use the registry object directly here and in other land types.